packages/runtime-common/index-writer.ts

```diff
@@ -222,6 +233,40 @@ export class Batch {
     return query(this.dbAdapter, expression, coerceTypes);
   }
 
+  private async getProductionVersion(url: URL) {
+    let [entry] = (await this.query([
+      `SELECT i.*`,
+      `FROM boxel_index as i
+       INNER JOIN realm_versions r ON i.realm_url = r.realm_url
+       WHERE`,
+      ...every([
+        any([
+          [`i.url =`, param(url.href)],
+          [`i.file_alias =`, param(url.href)],
+        ]),
+        realmVersionExpression(),
+      ]),
+    ] as Expression)) as unknown as BoxelIndexTable[];
+    if (!entry) {
+      return undefined;
+    }
+
+    let {
+      indexed_at: _remove1,
```

Comment:
What is _removeN and where do they come from?

Reply:
it's a simple way to use object destructuring to delete properties from an object. This way it we don't mutate a type by deleting a property